packages/react/src/createReactQueryHooks.tsx

```diff
@@ -62,8 +62,12 @@ export interface UseTRPCInfiniteQueryOptions<TPath, TInput, TOutput, TError>
     >,
     TRPCUseQueryBaseOptions {}
 
-export interface UseTRPCMutationOptions<TInput, TError, TOutput>
-  extends UseMutationOptions<TOutput, TError, TInput>,
+export interface UseTRPCMutationOptions<
+  TInput,
+  TError,
+  TOutput,
+  TContext = unknown,
```

Comment:
Let's restrict it to a `Record<string, unknown>`

```suggestion
  TContext extends Record<string, unknown> = {},
```

Reply:
I object to that.
Original type of `TContext` in `react-query` doesn't extend anything and I personally use primitive there sometimes, so I think there should be no restriction like that.